### PR TITLE
feat: configure route in home logo

### PR DESCRIPTION
**Pull request by BLasan**
## Description
Added routing functionality from the logo and text.

## Related issues and discussion
#717 

## Screenshots, if any
web-app1

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

**Comment by BLasan** [comment]
> Can you modify the commit message for this, please? there should be a space after feat:, also it s Fixes not Fix in the commit message, `feat: configure route in home logo (#718)`

Sorry. I'll update the commit now :)

**Comment by BLasan** [comment]
> Can you modify the commit message for this, please? there should be a space after feat:, also it s Fixes not Fix in the commit message, `feat: configure route in home logo (#718)`

It's done. Please have a look

File: src/app/core/shell/sidenav/sidenav.component.scss [hunk 1]
```diff
@@ -176,3 +176,7 @@
     }
   }
 }
+
```
Comment (karantakalkar) [inline src/app/core/shell/sidenav/sidenav.component.scss:179]: Remove this.

Comment (karantakalkar) [reply src/app/core/shell/sidenav/sidenav.component.scss:179]: replace app.logo at line 135 with :
```suggestion
.app-logo {
      margin: auto;
      &:hover {
        cursor: pointer;
      }
   }
```

Comment (BLasan) [reply src/app/core/shell/sidenav/sidenav.component.scss:179]: > replace app.logo at line 135 with :

The new class should be started at line 179? Or just replace this code with the previous one without changing the position?

Comment (karantakalkar) [reply src/app/core/shell/sidenav/sidenav.component.scss:179]: No just add 
&:hover {
        cursor: pointer;
      }
to app.logo at line 135 and remove the new homeIcon id youve created

File: src/app/core/shell/sidenav/sidenav.component.scss [hunk 2]
```diff
@@ -176,3 +178,6 @@
     }
   }
 }
+#homeIcon:hover {
```
Comment (karantakalkar) [inline src/app/core/shell/sidenav/sidenav.component.scss:181]: remove this

File: src/app/core/shell/sidenav/sidenav.component.scss [hunk 3]
```diff
@@ -134,8 +134,10 @@
   .app-brand {
     .app-logo {
       margin: auto;
+      &:hover {
+        cursor: pointer;
+      }
     }
-
     .app-logo-text {
```
Comment (karantakalkar) [inline src/app/core/shell/sidenav/sidenav.component.scss:141]: dont remove this line

File: src/app/core/shell/sidenav/sidenav.component.html [hunk 4]
```diff
@@ -7,7 +7,7 @@
   <div class="navigation-panel" fxLayout="column">
 
     <!-- TODO: Allow customization by organization -->
-    <div class="app-brand">
+    <div class="app-brand" [routerLink]="['/home']" id="homeIcon">
```
Comment (karantakalkar) [inline src/app/core/shell/sidenav/sidenav.component.html:10]: remove id="homeIcon"

File: src/app/core/shell/sidenav/sidenav.component.scss (hunk 3 above)
Comment (BLasan) [reply src/app/core/shell/sidenav/sidenav.component.scss:142]: > dont remove this line (138)

Left 141 as a blank line.

Comment (BLasan) [reply src/app/core/shell/sidenav/sidenav.component.scss:142]: Please review if the required change has been done properly

**Pull request merged**